trieste/models/gpflow/architectures.py

```diff
@@ -0,0 +1,401 @@
+# Copyright 2021 The Trieste Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" This file contains builders for GPflow models supported in Trieste. """
```

Comment:
Why is this file called architectures (rather than builders/factories/etc)?

Reply:
good point, I guess bcs in DGPs and Keras we called the file architectures - for Keras perhas that name makes the most sense while here and and for gpflux it could be renamed to builders.py?
(there is a single builder in keras architecture file, but this could be moved to its own file...)

Reply:
we opted for builders